### PR TITLE
Publish images also on Github Packages

### PR DESCRIPTION
**Pull request by ger-benjamin**
For  #1096 and [GSPOE-39](https://jira.camptocamp.com/browse/GSPOE-39)

- Rename camptocamp/pyramid-oereb to openoereb/pyramid-oereb internally in the project.
- Still push openoereb/pyramid-oereb to DockerHub camptopcamp/pyramid-oereb for now.
- Push openoereb/pyramid-oereb to Github Packages ghcr.io/openoereb/pyramid-oereb/pyramid-oereb

### CODE REVIEW

**Review by jwkaltz — APPROVED** [review]
LGTM - maybe @danduk82 should also have a look.

File: travis/deploy-docker.sh
Comment (ger-benjamin) [inline travis/deploy-docker.sh:4]: oups, need credential here

**Comment by ger-benjamin** [comment]
Currently I've a `denied: permission_denied: create_package` If I push the image manually.
I'll not merge before to be able to merge manually

**Comment by ger-benjamin** [comment]
Now I can push.
I had to add the c2c-bot-gis-ci into the openoereb **organization**.
Adding it into the pyramid-oereb project with write permissions was not sufficent. Probably useless in fact. I've removed it from this project and It looks that I can still push images.

**Comment by ger-benjamin** [comment]
I merge to see if the daily release will work. But I still would like a review if possible @danduk82

**Comment by ger-benjamin** [comment]
Looks good:
DockerHub - Last update: Dec 7, 2020 at 8:34 pm
Github Packages - Last update: Dec 7, 2020 at 8:35 pm

**Pull request merged**
